forta-bots/L1-Teleport-Backing/PoC/TeleportJoin.sol

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
```

Comment:
why sol files are dumped here?

Reply:
This contracts are the ones we used as PoC for testing the bots. They have tiny modifications to allow us to easily change some values in the contract so we can test the bots. Each time we create a PoC like this one we include the contracts in the repo, but if you want to, we can delete them.